Tools/samples.h

```diff
@@ -74,8 +74,8 @@ namespace AnaSamples
         }
 
         const T& null() const {return nullT_;}
-    protected:
         std::map<std::string, T> sampleSet_;
+    protected:
```

Comment:
The baseclass implements the [] operator to retrieve a single sample by name and provides begin() and end() to allow iteration over the collection.  How do you access the class?